### ISSUE TITLE
Git-2a634e4 not build (or not saved ?) ProcessHacker.pdb

### ISSUE THREAD

**Issue by VictorVG** [processhacker/processhacker#246]
The I use my current build toolkit I have error because ProcessHacker.pdb does exists:

![1](https://user-images.githubusercontent.com/906886/37237237-110097c2-2422-11e8-98e9-b4047bc1bf17.png)

I will correct my toolkit, but "ProcessHacker.pdb" is necessary for searching for errors and its saving is desirable.

**Comment by VictorVG** [comment]
The question has arisen - and how correctly to transfer CustomBuildTool. Its parametre-flag of formation of .PDB files for configurations Release and Release64? I tried the versions "-release BuildDebug=True", "-release" /p: ExtendedBuildDebug=True", " -release" "-BuildDebug=True", "BuildDebug=True", "-release /DEBUG". "-release BuildDebug" and finally got confused with what key should be passed in the command line, although I tried to understand the source code and what the utility is waiting for? As a result, I corrected the project file for the test assembly.:)
